src/utilities/util_printlog.c

```diff
@@ -28,11 +28,14 @@ util_printlog(WT_SESSION *session, int argc, char *argv[])
 
     flags = 0;
     ofile = NULL;
-    while ((ch = __wt_getopt(progname, argc, argv, "f:x")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "f:mx")) != EOF)
         switch (ch) {
         case 'f': /* output file */
             ofile = __wt_optarg;
             break;
+        case 'm': /* messages only */
+            LF_SET(WT_TXN_PRINTLOG_MSG);
+            break;
```

Comment:
Not related exactly but while in the neighborhood I've also thought it would be useful to have a start/end designation to printlog so that one could only print a subset of the log.

Reply:
In general I agree, something that would allow us to specify by LSN and/or timestamp. I've created a ticket WT-6309 to track the change.